truffle.js

```diff
@@ -5,6 +5,13 @@ module.exports = {
             port: 8545,
             network_id: "*" // Match any network id
         },
+        coverage: {
+            host: "localhost",
+            network_id: "*",
+            port: 8555,
+            gas: 0xfffffffffff,
+            gasPrice: 0x01
```

Comment:
Yeah, not until Metropolis possibly, because of a new opcode which will allow delegate calls to the contract, avoiding expensive copy ops for new instances.